modules/removebook.js

```diff
@@ -0,0 +1,10 @@
+/* eslint import/prefer-default-export: "off" */
+import { Book } from './book.js';
+
+function removeBook(index) {
+  const removeBooks = new Book();
+  removeBooks.deleteBook(index);
+  window.onload = removeBooks.displayBooks();
+}
```

Comment:
- [ ]  As we must migrate our code towards ES6 syntax, kindly refactor this function in order to use `arrow` syntax.